openstack/compute/v2/extensions/remoteconsoles/requests.go

```diff
@@ -0,0 +1,83 @@
+package remoteconsoles
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// RemoteConsoleProtocol represents valid remote console protocol.
+// It can be used to create a remote console with one of the pre-defined protocol.
+type RemoteConsoleProtocol string
+
+const (
+	// RemoteConsoleVNCProtocol represents the VNC console protocol.
+	RemoteConsoleVNCProtocol RemoteConsoleProtocol = "vnc"
+
+	// RemoteConsoleSPICEProtocol represents the SPICE console protocol.
+	RemoteConsoleSPICEProtocol RemoteConsoleProtocol = "spice"
+
+	// RemoteConsoleRDPProtocol represents the RDP console protocol.
+	RemoteConsoleRDPProtocol RemoteConsoleProtocol = "rdp"
+
+	// RemoteConsoleSerialProtocol represents the serial console protocol.
+	RemoteConsoleSerialProtocol RemoteConsoleProtocol = "serial"
+
+	// RemoteConsoleMKSProtocol represents the MKS console protocol.
+	RemoteConsoleMKSProtocol RemoteConsoleProtocol = "mks"
+)
+
+// RemoteConsoleType represents valid remote console type.
+// It can be used to create a remote console with one of the pre-defined type.
+type RemoteConsoleType string
+
+const (
+	// RemoteConsoleNoVNCType represents the VNC console type.
+	RemoteConsoleNoVNCType RemoteConsoleType = "novnc"
```

Comment:
Similar here: `ConsoleTypeNoVNC`. Same pattern for the rest, where the unique part is at the end of the name.